### DISCUSSION TITLE
In light of the announcement

### DISCUSSION THREAD

**Discussion by Wandering-Heiho** [dackst/nayuta#16] [General]
With NIS America having announced a localization of Nayuta, I once again want to thank you for taking the time and effort in making this improved patch of the PSP version. This is a day I wasn't sure would ever come.

I know I'm bad in trying to explain my feelings about this without coming off as... selfish? I'm not quite sure the term I'm looking for. Nevertheless, I enjoyed the time I spent in helping with this patch, whether it being from proofreading or altering the movie files frame by frame. I do not regret the time I spent doing any of it.

In the end, I hope you too enjoyed the time you spent making this patch as great as it is. I know I don't have enough information regarding whether you were annoyed in making this patch, or if you ever had days that you just wanted to scrap this project, but I hope you have a sense of accomplishment in what you were able to make.

From the bottom of my heart. thank you for making it possible to better enjoy the story and dialogue of this game.

**Comment by dackst** [comment]
There were some tiny regrets and unexpected annoyances along the way, but thanks, and thanks for the help in making it even better.
